doc/source/getting_started.rst

```diff
@@ -7,82 +7,47 @@ Getting started
 Installation
 ------------
 
-.. raw:: html
+.. panels::
+    :card: + install-card
+    :column: col-lg-6 col-md-6 col-sm-12 col-xs-12 p-3
 
-    <div class="container">
-        <div class="row">
-            <div class="col-lg-6 col-md-6 col-sm-12 col-xs-12 d-flex install-block">
-                <div class="card install-card shadow w-100">
-                <div class="card-header">
-                    Working with conda?
-                </div>
-                <div class="card-body">
-                    <p class="card-text">
+    Working with conda?
+    ^^^^^^^^^^^^^^^^^^^
 
-SciPy is part of the `Anaconda <https://docs.continuum.io/anaconda/>`__ distribution and can be
-installed with Anaconda or Miniconda:
+    SciPy is part of the `Anaconda <https://docs.continuum.io/anaconda/>`__
+    distribution and can be installed with Anaconda or Miniconda:
 
-.. raw:: html
+    ++++++++++++++++++++++
 
-                    </p>
-                </div>
-                <div class="card-footer text-muted">
+    .. code-block:: bash
 
-.. code-block:: bash
+        conda install scipy
 
-   conda install scipy
+    ---
 
-.. raw:: html
+    Prefer pip?
+    ^^^^^^^^^^^
 
-                </div>
-                </div>
-            </div>
-            <div class="col-lg-6 col-md-6 col-sm-12 col-xs-12 d-flex install-block">
-                <div class="card install-card shadow w-100">
-                <div class="card-header">
-                    Prefer pip?
-                </div>
-                <div class="card-body">
-                    <p class="card-text">
+    SciPy can be installed via pip from `PyPI <https://pypi.org/project/scipy>`__.
 
-SciPy can be installed via pip from `PyPI <https://pypi.org/project/scipy>`__.
+    ++++
 
-.. raw:: html
+    .. code-block:: bash
 
-                    </p>
-                </div>
-                <div class="card-footer text-muted">
+        pip install scipy
 
-.. code-block:: bash
+    ---
+    :column: col-12 p-3
 
-   pip install scipy
+    In-depth instructions?
+    ^^^^^^^^^^^^^^^^^^^^^^
 
-.. raw:: html
+    Installing a specific version? Installing from source? Check the advanced
+    installation page.
 
-                </div>
-                </div>
-            </div>
-            <div class="col-12 d-flex install-block">
-                <div class="card install-card shadow w-100">
-                <div class="card-header">
-                    In-depth instructions?
-                </div>
-                <div class="card-body">
-                    <p class="card-text">Installing a specific version?
-                      Installing from source?
-                      Check the advanced installation page.</p>
+    .. container:: custom-button
```

Comment:
This appears not to have worked:

https://29663-1460385-gh.circle-artifacts.com/0/html-scipyorg/getting_started.html

Presumably because you killed the `custom-button` CSS below. Probably you want to use `.. link-button::` here, too?

Reply:
Indeed, thanks!